libs/ui-components/src/components/Device/EditDeviceWizard/utils.ts

```diff
@@ -35,7 +35,8 @@ export const getDevicePatches = (currentDevice: Device, updatedDevice: EditDevic
 
   // Device labels
   const currentLabels = currentDevice.metadata.labels || {};
-  const updatedLabels = updatedDevice.labels || [];
+  const updatedLabels = [...updatedDevice.labels];
```

Comment:
Form values were being modified and the "alias" label would be duplicate when there was a failure in submitting the form.